src/stk/databases/mongo_db/value.py

```diff
@@ -98,21 +101,49 @@ def __init__(
             key, they will return the same value from the database.
 
         lru_cache_size : :class:`int`, optional
```

Comment:
I like that this is being depreciated, rather than remove outwrite.

Reply:
Yep, my goal is to have a relatively stable API from now. So any API change affecting users will ideally have at least a 6 month warning.